.github/workflows/ci.yml

```
@@ -0,0 +1,43 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - actions/checkout@v2.3.4
+      - actions/setup-node@v2.1.2
+      - yarn install --frozen-lockfile
+      - yarn run lint
+
+  test:
+    strategy:
+      fail-fast: false
+      matrix:
+        node:
+          - "15"
+          - "14"
+          - "10"
```

We should probably test our exact minimal supported version:

```suggestion
          - "10.3.0"
```